frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -150,11 +151,13 @@
         // API call to manage the worker from UI.
         // Response data will be like: {action: "Success" or "Failure", error: <String to include only if action is Failure.>}
         vm.manageWorker = function(action){
+            if (action == "start" || action == "restart") vm.disableWorkerButtons = true;
```

Comment:
@savish28 why are we not checking for stop action?

Reply:
@Ram81 because the motivation behind this PR is this suggestion https://github.com/Cloud-CV/EvalAI/pull/2918#issuecomment-677794344.

Reply:
We can add the stop action as well if you suggest. I thought that Starting and Restarting are heavy operations compared to stopping.

Reply:
@savish28 I think it was not clear in that issue but the idea is to disable the start/stop/restart buttons based on status of the workers that will be fetched using boto3 API calls. Idea is to fetch status of all running workers for a challenge and use the status of those workers to disable these action buttons. So this PR will need changes which are dependent on this #3308 PR. Once it is merged you can make the required changes.